packages/db/postgres/client.js

```diff
@@ -20,7 +20,8 @@ const uploadQuery = `
         content(cid, dagSize:dag_size, pins:pin(status, updated:updated_at, location:pin_location(_id:id, peerId:peer_id, peerName:peer_name, region)))
       `
 /**
- * @typedef {import('./postgres/pg-rest-api-types').definitions} definitions
+ * @typedef {import('./pg-rest-api-types.js').definitions} definitions
+ * @typedef {import('@supabase/postgrest-js').PostgrestError} PostgrestError
```

Comment:
It's for typecasting as described [here](https://docs.joshuatz.com/cheatsheets/js/jsdoc/#simple-casting)

Actually in this particular instance is not really needed since the problem is that `data` is not in the typedef,  but  I've made a similar change it in 2 different instances in the file ([here](https://github.com/web3-storage/web3.storage/pull/611/files#diff-04117e09fe44d5643c6812b42b94359638abe1e2c2e1e2ba64065c6ca3228819R107) and [here](https://github.com/web3-storage/web3.storage/pull/611/files#diff-04117e09fe44d5643c6812b42b94359638abe1e2c2e1e2ba64065c6ca3228819R356))

And I've realized, instead of casting we should be calling 
`.single()` since in all those places we're actually expecting a single result, isn't it?

`.rpc()` return `PostgrestFilterBuilder<any>` while we'd be expecting is PostgrestSingleResponse<any>

What do you think?